cpp/src/ray/util/process_helper.cc

```diff
@@ -119,14 +103,18 @@ void ProcessHelper::RayStart(CoreWorkerOptions::TaskExecutionCallback callback)
   RAY_CHECK(!ConfigInternal::Instance().plasma_store_socket_name.empty());
   RAY_CHECK(ConfigInternal::Instance().node_manager_port > 0);
 
-  auto session_dir = ConfigInternal::Instance().session_dir.empty()
-                         ? GetSessionDir(redis_ip, ConfigInternal::Instance().redis_port,
-                                         ConfigInternal::Instance().redis_password)
-                         : ConfigInternal::Instance().session_dir;
-
-  auto log_dir = ConfigInternal::Instance().logs_dir.empty()
-                     ? session_dir + "/logs"
-                     : ConfigInternal::Instance().logs_dir;
+  std::string log_dir = ConfigInternal::Instance().logs_dir;
```

Comment:
the log_dir is a temporary, it can be moved later.

Reply:
ok